appveyor.yml

```diff
@@ -5,7 +5,18 @@
 #---------------------------------#
 
 # version format
-version: 1.1.0.{build}
+version: 1.2.0.{build}-ci
+
+for:
+- branches:
+    only:
+      - master
+  version: 1.2.0.{build}-rc

```

Comment:
behaves not exactely as intended. Already the PR build got the `rc` indicator - intended was only after merge. But that seems to be an GitHub issue. Was the appveyor build created as git or GitHub build?

Reply:
no, the build Format is set in the `appveyor.yml` This redefines your Settings. It's an Github issue - GitHub merges PRs virtually already to master and "put in production", so called Github flow.
This is a minor Thing, Nothing you can or need to do here.